### PR TITLE
Improve RPC documentation

### PR DESCRIPTION
**Pull request by olaurendeau**

### CODE REVIEW

File: src/Swarrot/Processor/RPC/README.md [hunk 1]
```diff
@@ -3,24 +3,38 @@ The RPC processors are [swarrot](https://github.com/swarrot/swarrot)'s processor
 Their goal is to handle RPC communications between a RPC client and a RPC server.
 Their usage is not mutually inclusive ; you can run one without the other.
 
+`reply_to` message property contain the name of the queue on which the server will send the response, if exchange name is empty broker will receiver the message on the default exchange and route on the queue defined in the routing_key
```
Comment (odolbeau) [inline src/Swarrot/Processor/RPC/README.md]: will receive

File: src/Swarrot/Processor/RPC/README.md [hunk 2]
```diff
@@ -3,24 +3,38 @@ The RPC processors are [swarrot](https://github.com/swarrot/swarrot)'s processor
...
 First of all, you'll need to declare an exchange (let's say a `rpc` exchange),
 and a queue from which the server processor will consume from, with an appropriate
 routing key. Let's call the queue `rpc_fibonacci`, and the routing key `fibonacci`.
 
-On the server part, instantiate a new Consumer with a Server Processor ; on the
-client part, send a message to the `rpc_fibonacci` queue (through the `rpc`
-exchange), and then launch a consumer with the client processor, in which you'll
-receive the response from the server ; the client is waiting for the server to 
-complete its designated task.
+Bind queue `rpc_fibonacci` to exchange `rpc` with routing key `fibonacci`.
```
Comment (odolbeau) [inline src/Swarrot/Processor/RPC/README.md]: Maybe don't mention the routing_key as the default behaviour is to let it empty?

Comment (olaurendeau) [inline src/Swarrot/Processor/RPC/README.md]: For the request you can use any kind of exchange / binding / queue :)
The `reply_to` feature is only for the response

Comment (odolbeau) [inline src/Swarrot/Processor/RPC/README.md]: Makes sense! :)

**Pull request merged**